auction-app-frontend/src/pages/Shop/Shop.jsx

```diff
@@ -34,51 +36,76 @@ const Shop = () => {
   const [checked, setChecked] = useState({});
   const [selectedSorting, setSelectedSorting] = useState(SHOP_PAGE_SORTING[0]);
   const [sortingDirection, setSortingDirection] = useState(selectedSorting.direction);
+  
+  const [initialMinPrice, setInitialMinPrice] = useState(0);
+  const [initialMaxPrice, setInitialMaxPrice] = useState(100);
```

Comment:
Why initial and actual price?

Reply:
initial represents the max and min price that user can select, if i kept the same state for initial and actual price i wouldnt be able to set the "limit" once user selects a range